--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-prx6hd	2012-09-07 13:55:53.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-u1df7z	2012-09-07 13:55:53.000000000 -0700
@@ -3,8 +3,8 @@
 Number.abs(-0) returns a int = int PASSED!
 Number.abs(1) returns a int = int PASSED!
 Number.abs() length is 1 = 1 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.abs() with no args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.abs() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.abs(undefined) = NaN PASSED!
 Number.abs(string) = NaN PASSED!
 Number.abs(NaN) = NaN PASSED!
@@ -32,11 +32,11 @@
 Number.abs('1') = 1 PASSED!
 Number.abs('0') = 0 PASSED!
 Number.NEGATIVE_INFINITY/Number.abs('0') = Infinity PASSED!
-Number.abs(-Number.MIN_VALUE) = 4.9406564584124654e-324 PASSED!
-Number.abs(-Number.MAX_VALUE) = 1.79769313486231e+308 PASSED!
-Number.abs(Number.MIN_VALUE) = 4.9406564584124654e-324 PASSED!
-Number.abs(Number.MAX_VALUE) = 1.79769313486231e+308 PASSED!
-Number.abs(-Number.MIN_VALUE) check() = 4.9406564584124654e-324 PASSED!
-Number.abs(-Number.MAX_VALUE) check() = 1.79769313486231e+308 PASSED!
-Number.abs(Number.MIN_VALUE) check() = 4.9406564584124654e-324 PASSED!
-Number.abs(Number.MAX_VALUE) check() = 1.79769313486231e+308 PASSED!
+Number.abs(-Number.MIN_VALUE) = 5e-324 PASSED!
+Number.abs(-Number.MAX_VALUE) = 1.7976931348623157e+308 PASSED!
+Number.abs(Number.MIN_VALUE) = 5e-324 PASSED!
+Number.abs(Number.MAX_VALUE) = 1.7976931348623157e+308 PASSED!
+Number.abs(-Number.MIN_VALUE) check() = 5e-324 PASSED!
+Number.abs(-Number.MAX_VALUE) check() = 1.7976931348623157e+308 PASSED!
+Number.abs(Number.MIN_VALUE) check() = 5e-324 PASSED!
+Number.abs(Number.MAX_VALUE) check() = 1.7976931348623157e+308 PASSED!
