--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-wx71ak	2012-09-07 00:30:39.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-qcs6h8	2012-09-07 00:30:39.000000000 -0700
@@ -2,7 +2,7 @@
 Number.acos() returns a Number = Number PASSED!
 Number.acos() length is 1 = 1 PASSED!
 Asserting for ArgumentError = true PASSED!
-Number.acos() with no args = ArgumentError: Error #1063 PASSED!
+Number.acos() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.acos(undefined) = NaN PASSED!
 Number.acos(string) = NaN PASSED!
 Number.acos(NaN) = NaN PASSED!
