--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-1ur55wu	2012-09-07 13:55:53.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-vejzmx	2012-09-07 13:55:53.000000000 -0700
@@ -1,8 +1,8 @@
 15.8.2.3 public native static function asin (x:Number) :Number;
 Number.asin() returns a Number = Number PASSED!
 Number.asin() length is 1 = 1 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.asin() with no args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.asin() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.asin(undefined) = NaN PASSED!
 Number.asin(string) = NaN PASSED!
 Number.asin(NaN) = NaN PASSED!
@@ -31,5 +31,5 @@
 Number.asin(0) NumberLiteral = 0 PASSED!
 Number.asin(-0) NumberLiteral = 0 PASSED!
 Number.asin(-1) NumberLiteral = -1.5707963267948966 PASSED!
-Number.asin(Number.SQRT1_2) = 0.7853981633974484 PASSED!
-Number.asin(-Number.SQRT1_2) = -0.7853981633974484 PASSED!
+Number.asin(Number.SQRT1_2) = 0.7853981633974485 PASSED!
+Number.asin(-Number.SQRT1_2) = -0.7853981633974485 PASSED!
