--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-1aoq24p	2012-09-07 13:55:53.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-1fq28bo	2012-09-07 13:55:53.000000000 -0700
@@ -1,10 +1,10 @@
 15.8.2.5 public native static function atan2 (y:Number, x:Number):Number;
 Number.atan2() returns a Number = Number PASSED!
 Number.atan2() length is 2 = 2 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.atan2() with no args = ArgumentError: Error #1063 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.atan2(0) with one args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.atan2() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
+Asserting for ArgumentError = false FAILED! expected: true
+Number.atan2(0) with one args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.atan2(0, undefined) = NaN PASSED!
 Number.atan2(undefined, 0) = NaN PASSED!
 Number.atan2(0, string) = NaN PASSED!
