--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-fohv6r	2012-09-07 13:55:53.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-8uuqx1	2012-09-07 13:55:53.000000000 -0700
@@ -1,8 +1,8 @@
 15.8.2.4 public native static function atan (x:Number) :Number;
 Number.atan() returns a Number = Number PASSED!
 Number.atan() length is 1 = 1 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.atan() with no args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.atan() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.atan(undefined) = NaN PASSED!
 Number.atan(string) = NaN PASSED!
 Number.atan(NaN) = NaN PASSED!
