--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-16n2dlj	2012-09-07 00:30:39.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-4z1fif	2012-09-07 00:30:39.000000000 -0700
@@ -2,7 +2,7 @@
 Number.ceil() returns a int = int PASSED!
 Number.ceil() length is 1 = 1 PASSED!
 Asserting for ArgumentError = true PASSED!
-Number.ceil() with no args = ArgumentError: Error #1063 PASSED!
+Number.ceil() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.ceil(undefined) = NaN PASSED!
 Number.ceil(string) = NaN PASSED!
 Number.ceil(NaN) = NaN PASSED!
@@ -33,7 +33,7 @@
 Number.ceil(null) = 0 PASSED!
 Number.ceil(true) = 1 PASSED!
 Number.ceil(false) = 0 PASSED!
-Number.ceil(Number.MAX_VALUE) = 1.79769313486231e+308 PASSED!
+Number.ceil(Number.MAX_VALUE) = 1.7976931348623157e+308 PASSED!
 Number.ceil(Number.MAX_VALUE+1.79769313486231e+308) = Infinity PASSED!
 Number.ceil(Number.MIN_VALUE) = 1 PASSED!
 Number.ceil('1') = 1 PASSED!
