--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-1fe8ur9	2012-09-07 00:30:39.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-1kukpq4	2012-09-07 00:30:39.000000000 -0700
@@ -2,7 +2,7 @@
 Number.cos() returns a Number = Number PASSED!
 Number.cos() length is 1 = 1 PASSED!
 Asserting for ArgumentError = true PASSED!
-Number.cos() with no args = ArgumentError: Error #1063 PASSED!
+Number.cos() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.cos(undefined) = NaN PASSED!
 Number.cos(string) = NaN PASSED!
 Number.cos(NaN) = NaN PASSED!
