--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-zezltr	2012-09-07 00:30:39.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-1xnbvgq	2012-09-07 00:30:39.000000000 -0700
@@ -2,7 +2,7 @@
 Number.exp() returns a Number = Number PASSED!
 Number.exp() length is 1 = 1 PASSED!
 Asserting for ArgumentError = true PASSED!
-Number.exp() with no args = ArgumentError: Error #1063 PASSED!
+Number.exp() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.exp(undefined) = NaN PASSED!
 Number.exp(string) = NaN PASSED!
 Number.exp(NaN) = NaN PASSED!
