--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-5s83gr	2012-09-07 13:55:53.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-1b0fkvo	2012-09-07 13:55:53.000000000 -0700
@@ -1,8 +1,8 @@
 15.8.2.8 public native static function exp (x:Number) :Number;
 Number.exp() returns a Number = Number PASSED!
 Number.exp() length is 1 = 1 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.exp() with no args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.exp() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.exp(undefined) = NaN PASSED!
 Number.exp(string) = NaN PASSED!
 Number.exp(NaN) = NaN PASSED!
