--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-ymzjmq	2012-09-07 13:55:53.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-1p3jwkg	2012-09-07 13:55:53.000000000 -0700
@@ -1,8 +1,8 @@
 15.8.2.9 public native static function floor (x:Number) :Number;
 Number.floor() returns a int = int PASSED!
 Number.floor() length is 1 = 1 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.floor() with no args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.floor() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.floor(undefined) = NaN PASSED!
 Number.floor(string) = NaN PASSED!
 Number.floor(NaN) = NaN PASSED!
@@ -34,7 +34,7 @@
 Number.floor(true) = 1 PASSED!
 Number.floor('1') = 1 PASSED!
 Number.floor(-Number.MIN_VALUE) = -1 PASSED!
-Number.floor(Number.MAX_VALUE) = 1.79769313486231e+308 PASSED!
+Number.floor(Number.MAX_VALUE) = 1.7976931348623157e+308 PASSED!
 Number.floor(myNum=1) = 1 PASSED!
 Number.floor(myNum=0) = 0 PASSED!
 Number.INFINITY/Number.floor(myNum=0) = Infinity PASSED!
