--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-ifvqib	2012-09-07 00:30:39.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-rte0ur	2012-09-07 00:30:39.000000000 -0700
@@ -2,7 +2,7 @@
 Number.log() returns a Number = Number PASSED!
 Number.log() length is 1 = 1 PASSED!
 Asserting for ArgumentError = true PASSED!
-Number.log() with no args = ArgumentError: Error #1063 PASSED!
+Number.log() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.log(undefined) = NaN PASSED!
 Number.log(string) = NaN PASSED!
 Number.log(NaN) = NaN PASSED!
