--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-w1dmxw	2012-09-07 13:55:53.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-1e71hyn	2012-09-07 13:55:53.000000000 -0700
@@ -1,8 +1,8 @@
 15.8.2.10 public native static function log(x:Number):Number;
 Number.log() returns a Number = Number PASSED!
 Number.log() length is 1 = 1 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.log() with no args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.log() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.log(undefined) = NaN PASSED!
 Number.log(string) = NaN PASSED!
 Number.log(NaN) = NaN PASSED!
