--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-1xxc3w6	2012-09-07 13:55:53.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-192g06f	2012-09-07 13:55:53.000000000 -0700
@@ -1,5 +1,5 @@
 15.8.2.14 public native static function random ():Number;
 Number.random() returns a Number = Number PASSED!
 Number.random() length is 0 = 0 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.random() with args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.random() with args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
