--- /private/var/folders/mx/sq278bz91b5bwgl3myhqmpw00000gn/T/out11271-98635-z06ai8	2012-08-27 16:16:06.000000000 -0700
+++ /private/var/folders/mx/sq278bz91b5bwgl3myhqmpw00000gn/T/out11271-98635-feuro9	2012-08-27 16:16:06.000000000 -0700
@@ -1,8 +1,8 @@
 15.8.2.16 public native static function sin (x:Number) :Number;
 Number.sin() returns a Number = Number PASSED!
 Number.sin() length is 1 = 1 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.sin() with no args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.sin() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.sin(undefined) = NaN PASSED!
 Number.sin(string) = NaN PASSED!
 Number.sin(NaN) = NaN PASSED!
