--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-10ysz9a	2012-09-07 13:55:53.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-157ctvg	2012-09-07 13:55:53.000000000 -0700
@@ -1,8 +1,8 @@
 15.8.2.17 public native static function sqrt (x:Number) :Number;
 Number.sqrt() returns a Number = Number PASSED!
 Number.sqrt() length is 1 = 1 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.sqrt() with no args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.sqrt() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.sqrt(undefined) = NaN PASSED!
 Number.sqrt(string) = NaN PASSED!
 Number.sqrt(NaN) = NaN PASSED!
