--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-vz34uw	2012-09-07 00:30:39.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-tfjfkj	2012-09-07 00:30:39.000000000 -0700
@@ -2,7 +2,7 @@
 Number.sqrt() returns a Number = Number PASSED!
 Number.sqrt() length is 1 = 1 PASSED!
 Asserting for ArgumentError = true PASSED!
-Number.sqrt() with no args = ArgumentError: Error #1063 PASSED!
+Number.sqrt() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.sqrt(undefined) = NaN PASSED!
 Number.sqrt(string) = NaN PASSED!
 Number.sqrt(NaN) = NaN PASSED!
