--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-iiuzkw	2012-09-07 00:30:39.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-39959-19j9w6v	2012-09-07 00:30:39.000000000 -0700
@@ -2,7 +2,7 @@
 Number.tan() returns a Number = Number PASSED!
 Number.tan() length is 1 = 1 PASSED!
 Asserting for ArgumentError = true PASSED!
-Number.tan() with no args = ArgumentError: Error #1063 PASSED!
+Number.tan() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.tan(undefined) = NaN PASSED!
 Number.tan(string) = NaN PASSED!
 Number.tan(NaN) = NaN PASSED!
