--- /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-1874t0k	2012-09-07 13:55:53.000000000 -0700
+++ /private/var/folders/19/q3tltmvs74j1q_jjrqsj4tdr0000gp/T/out11285-54285-xfgccw	2012-09-07 13:55:53.000000000 -0700
@@ -1,8 +1,8 @@
 15.8.2.18 public native static function tan (x:Number) :Number;
 Number.tan() returns a Number = Number PASSED!
 Number.tan() length is 1 = 1 PASSED!
-Asserting for ArgumentError = true PASSED!
-Number.tan() with no args = ArgumentError: Error #1063 PASSED!
+Asserting for ArgumentError = false FAILED! expected: true
+Number.tan() with no args = No errors. None. Nada. All FAILED! expected: ArgumentError: Error #1063
 Number.tan(undefined) = NaN PASSED!
 Number.tan(string) = NaN PASSED!
 Number.tan(NaN) = NaN PASSED!
