diff -r 2b38aa82c803 configure.py
--- a/configure.py	Tue Jan 29 06:59:24 2013 -0800
+++ b/configure.py	Thu Mar 21 17:40:43 2013 -0700
@@ -108,6 +112,10 @@
             FLAGS += "-Werror -Wempty-body -Wno-logical-op -Wmissing-field-initializers -Wstrict-aliasing=3 -Wno-array-bounds -Wno-clobbered -Wstrict-overflow=0 -funit-at-a-time  "
             if (MAJOR_VERSION == 4 and MINOR_VERSION == 6): # 4.6
                 FLAGS += "-Wno-psabi -Wno-unused-variable -Wno-unused-but-set-variable "
+            if (MAJOR_VERSION == 4 and MINOR_VERSION == 7): # 4.7
+                FLAGS += "-Wno-unused-but-set-variable -Wno-narrowing "
+            if (MAJOR_VERSION == 4 and MINOR_VERSION == 8): # 4.8
+                FLAGS += "-Wno-unused-but-set-variable -Wno-narrowing "

 
     return FLAGS
 
