diff -r 2b38aa82c803 test/util/runtestBase.py
--- a/test/util/runtestBase.py	Tue Jan 29 06:59:24 2013 -0800
+++ b/test/util/runtestBase.py	Thu Mar 21 17:23:08 2013 -0700
@@ -1249,7 +1249,7 @@
                 self.compile_test(test)
                 if self.ascOutputOnly:
                     continue  # no .abc created
-                if not exists(testdir+".abc"):
+                if not exists(testdir+".abc") and self.aotsdk:
                     print("ERROR abc files %s.abc not created" % (testdir))
                     self.ashErrors.append("abc files %s.abc not created" % (testdir))
                     continue
