--- a/configure.py
+++ b/configure.py
@@ -104,6 +108,8 @@
             FLAGS += "-Werror -Wempty-body -Wno-logical-op -Wmissing-field-initializers -Wstrict-aliasing=3 -Wno-array-bounds -Wno-clobbered -Wstrict-overflow=0 -funit-at-a-time  "
             if (MAJOR_VERSION == 4 and MINOR_VERSION == 6): # 4.6
                 FLAGS += "-Wno-psabi -Wno-unused-variable -Wno-unused-but-set-variable "
+            if (MAJOR_VERSION == 4 and MINOR_VERSION == 7): # 4.7
+                FLAGS += "-Wno-unused-but-set-variable -Wno-narrowing "
 
     return FLAGS
 
